src/main/java/util/miscellaneous/Chatbot.java

```diff
@@ -0,0 +1,52 @@
+package util.miscellaneous;
+
+public interface Chatbot {
+    String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    String HORIZONTAL_LINE = "───────────────────────────────────────────────────────────────────────";
+    String GREETING_MSG_01 = " I have been waiting for you, Mister Winters.";
+    String GREETING_MSG_02 = " Forgive my manners, call me the Duke. Now to business.";
+    String ITEM_NOT_EXIST_MSG = "Ah...The task you choose doesn't exist on your list.";
+    String MARKED_MSG = " Aha! An interesting selection!";
+    String UNMARKED_MSG = " Ah... What a shame!";
+    String DELETE_MSG = " So you have chosen to delete this task!";
+    String GOODBYE_MSG = " Good day, then!";
+    String NO_TASK_MSG = " Sorry but you haven't input a task here.";
+    String NO_DATE_MSG = " Sorry but you haven't input a date here.";
+    String IO_ERROR_MSG = " Oh no! File IO error just occurred.";
+    String NO_PREVIOUS_RECORD = " There is currently no save file.";
+
+
```

Comment:
Additional blank lines could be removed to increase code quality and consistency.